### PR TITLE
Update for new copyPointZ API

### PR DESCRIPTION
**Pull request by chrisasc**

### CODE REVIEW

**Comment by SatjaSivcev** [comment]
Did you run the CI locally @chrisasc?

File: source/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp [hunk 1]
```diff
@@ -15,32 +15,26 @@ This example shows how to convert point cloud from ZDF file to OpenCV format, th
...
-        y,
-        z
-    };
-
-    template<Axis axis>
-    float getValue(const Zivid::PointXYZ &p);
-
-    template<>
-    float getValue<Axis::z>(const Zivid::PointXYZ &p)
+    float getValue(const Zivid::PointZ &p)
```
Comment (SatjaSivcev) [inline source/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp:18]: This is now specific to Z so I would call it getValueZ

File: source/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp [hunk 2]
```diff
@@ -15,32 +15,26 @@ This example shows how to convert point cloud from ZDF file to OpenCV format, th
...
 
-    template<Axis axis>
-    bool isLesserOrNan(const Zivid::PointXYZ &a, const Zivid::PointXYZ &b)
+    bool isLesserOrNan(const Zivid::PointZ &a, const Zivid::PointZ &b)
     {
-        return getValue<axis>(a) < getValue<axis>(b) ? true : std::isnan(getValue<axis>(a));
+        if(std::isnan(getValue(a)) && std::isnan(getValue(b)))
+        {
+            return false;
+        }
```
Comment (SatjaSivcev) [inline source/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp:28]: What case do you cover by adding this?

Comment (chrisasc) [reply source/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp:28]: Some compilers dont like to compare a NaN value against another NaN value. This was the error Torbjørn and I got when we were running the sample in debug mode. This little if statement solves it

File: source/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp (hunk 1 above)
Comment (chrisasc) [reply source/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp]: done

**Comment by chrisasc** [comment]
> Did you run the CI locally @chrisasc?

No, i only committed the PR, Torbjørn wrote the code an ran it.

**Pull request merged**